tests/test_device/test_ipu/test_ipu_runner.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import os.path as osp
+
+import pytest
+import torch
+import torch.nn as nn
+from torch.utils.data import Dataset
+
+from mmcv.runner import build_runner
+from mmcv.utils import IS_IPU_AVAILABLE
+
+if IS_IPU_AVAILABLE:
+    from mmcv.device.ipu import IPUDataLoader, runner
+
+skip_no_ipu = pytest.mark.skipif(
```

Comment:
It seems that this function is in many files, can we put it into mmcv.utils.testing?

Reply:
copy this usage from other tests